martha/martha_v3.js

```diff
@@ -44,6 +46,17 @@ const overridePencilsDownSeconds = (seconds) => {
     pencilsDownSeconds = seconds;
 };
 
+const secretManagerServiceClient = new SecretManagerServiceClient();
+
+async function getSecret(name) {
+    const [version] = await secretManagerServiceClient.accessSecretVersion({
```

Comment:
Is this pretty quick? (We're calling this twice on every execution that uses passports.) Is there any API rate limit that we need to be worried about? I ask because we could potentially fetch these on cloud function initialization (or lazy fetch and cache) in case the instance gets used more than once.